ocs_ci/ocs/utils.py

```diff
@@ -925,45 +956,110 @@ def run_must_gather(log_dir_path, image, command=None, cluster_config=None):
+    get_helper_pods_output(log_dir_path)
+
+
+def get_logs_ocp_mg_pods(log_dir_path):
+    """
+    Get logs from OCP Must Gather pods
+
+    Args:
+        log_dir_path (str): the path of copying the logs
+
+    """
+    from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_logs
+
+    namespaces = get_namespce_name_by_pattern(pattern="openshift-must-gather")
+    try:
+        for namespace in namespaces:
+            pods_mg_ns = get_all_pods(namespace=namespace)
+            for pod_mg_ns in pods_mg_ns:
+                log.info(
+                    f"*** ocp_mg_pod_name: {pod_mg_ns.name} ocp_mg_pod_namespace: {namespace} ***"
+                )
+
+                file_path_describe = os.path.join(
+                    log_dir_path, f"describe_ocp_mg_{pod_mg_ns.name}.log"
+                )
+                with open(file_path_describe, "w") as df:
+                    df.write(f"ocp mg pod describe:\n{pod_mg_ns.describe()}")
+                log.debug(f"ocp mg pod describe:\n{pod_mg_ns.describe()}")
+
+                ocp_mg_pod_logs = get_pod_logs(
+                    pod_name=pod_mg_ns.name, namespace=namespace, all_containers=True
+                )
+                file_path_describe = os.path.join(
+                    log_dir_path, f"log_ocp_mg_{pod_mg_ns.name}.log"
+                )
+                with open(file_path_describe, "w") as df:
+                    df.write(ocp_mg_pod_logs)
+                log.error(f"ocp mg pod logs:\n{ocp_mg_pod_logs}")
+    except Exception as e:
+        log.error(e)
+
+
+def get_helper_pods_output(log_dir_path):
+    """
+    Get the output of "oc describe mg-helper pods"
+
+    Args:
+        log_dir_path (str): the path of copying the logs
 
     """
     from ocs_ci.ocs.resources.pod import get_pod_obj, get_pod_logs
 
-    output_describe_mg_helper = ""
     helper_pods = get_pod_name_by_pattern(pattern="helper")
     for helper_pod in helper_pods:
         try:
             helper_pod_obj = get_pod_obj(
                 name=helper_pod, namespace=ocsci_config.ENV_DATA["cluster_namespace"]
             )
-            output_describe_mg_helper += (
-                f"****helper pod {helper_pod} describe****\n{helper_pod_obj.describe()}\n"
-                f"****helper pod {helper_pod} logs***\n{get_pod_logs(pod_name=helper_pod)}"
+
+            describe_helper_pod = helper_pod_obj.describe()
+            file_path_describe = os.path.join(
+                log_dir_path, f"describe_ocs_mg_helper_pod_{helper_pod_obj.name}.log"
+            )
+            with open(file_path_describe, "w") as df:
+                df.write(describe_helper_pod)
+            log.debug(
+                f"****helper pod {helper_pod} describe****\n{describe_helper_pod}\n"
+            )
+
+            log_helper_pod = get_pod_logs(pod_name=helper_pod)
+            file_path_describe = os.path.join(
+                log_dir_path, f"log_ocs_mg_helper_pod_{helper_pod_obj.name}.log"
```

Comment:
```suggestion
                log_dir_path, f"log_ocs_mg_helper_pod_{helper_pod}.log"
```

Reply:
I did not get you.. I want to get the pod name..

Reply:
helper_pod variable already contains pod name. so no need to hit helper_pod_obj

Reply:
done